tests/func/remote/test_http.py

```diff
@@ -0,0 +1,12 @@
+from dvc.main import main
+
+
+def test_incompatibility_with_no_traverse(dvc_repo, caplog):
+    main(["remote", "add", "http", "https://example.com"])
+
+    assert 0 != main(["status", "--remote", "http"])
+
+    assert (
```

Comment:
This test is incorrect, no_traverse is True by default, which is supported by HTTP.